test/dutchExchange.js

```diff
@@ -155,7 +166,7 @@ contract('DutchExchange', (accounts) => {
       0,
       2,
       1,
-      { from: seller1 }
+      { from: seller1 },
```

Comment:
@W3stside Does the code run for you, if you have the comma here? My compiler always complains about it. That is why I removed it, although the linter says, it should be there.